components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/utils/GitpodLogger.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.utils
+
+import com.intellij.openapi.diagnostic.logger
+
+class GitpodLogger {
+    companion object {
+        private val logger = logger<GitpodLogger>()
+
+        /**
+         * When running launch-dev-server.sh, only logs with levels higher than Info (e.g. Warn, Error)
+         * are displayed on terminal, that's why we should log Info as Warn during development.
+         */
+        private val shouldLogInfoAsWarn = System.getenv("JB_DEV").toBoolean()
```

Comment:
i think it is redundant, you can search for `gitpod: `

Reply:
I see, I usually look at log file.

Could you ask JB folks or goole? I'm pretty sure we can add some configs